File: Examples/Features/Shape/ConfigAdjustmentPoints.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Shape
{
    public class ConfigAdjustmentPoints : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];
            // Add a right arrow callout.
            GrapeCity.Documents.Excel.Drawing.IShape shape = worksheet.Shapes.AddShape(GrapeCity.Documents.Excel.Drawing.AutoShapeType.RightArrowCallout, 20, 20, 200, 100);

            GrapeCity.Documents.Excel.Drawing.IAdjustments adjustments = shape.Adjustments;
            adjustments[0] = 0.5;// arrow neck width
            adjustments[1] = 0.4;// arrow head width
            adjustments[2] = 0.5;// arrow head height
            adjustments[3] = 0.6;// text box width
        }

        public override bool IsNew
        {
            get
            {
                return true;
            }
        }

    }
}

File: Examples.VB/Features/Shape/ConfigAdjustmentPoints.vb
Namespace Features.Shape
    Public Class ConfigAdjustmentPoints
        Inherits ExampleBase

        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            ' Add a right arrow callout.
            Dim shape As IShape = worksheet.Shapes.AddShape(AutoShapeType.RightArrowCallout, 20, 20, 200, 100)

            Dim adjustments As IAdjustments = shape.Adjustments
            adjustments(0) = 0.5 ' arrow neck width
            adjustments(1) = 0.4 ' arrow head width
            adjustments(2) = 0.5 ' arrow head height
            adjustments(3) = 0.6 ' text box width
        End Sub

        Public Overrides ReadOnly Property IsNew As Boolean
            Get
                Return True
            End Get
        End Property

    End Class
End Namespace
